### PR TITLE
Week 2 Servlet doGet function

### PR DESCRIPTION
**Pull request by KyaCarrington**
Week 2 Step 1 doGet function that should open my website

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/newServlet.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+/** Servlet that returns HTML that contains the page view count. */
+@WebServlet("/mypage.html")
+public class newServlet extends HttpServlet {
+
+  private int pageViews = 0;
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.getWriter().println("<h1>Hello world!</h1>");
```
Comment (enriquejosepadilla) [inline portfolio/src/main/java/com/google/sps/servlets/newServlet.java:15]: technically the code and the docs from line 9 don't match. You aren't returning the page views, yet.

**Review by enriquejosepadilla — CHANGES_REQUESTED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/newServlet.java (hunk 1 above)
Comment (KyaCarrington) [reply portfolio/src/main/java/com/google/sps/servlets/newServlet.java:15]: I just copy and pasted the top half of the document from the example one and can delete that comment. Other than that does everything look okay?

**Review by KyaCarrington — COMMENTED** [review]
I deleted those two lines because I think that's what my previous feedback was referring to

**Review by enriquejosepadilla — APPROVED** [review]

**Review by KyaCarrington — COMMENTED** [review]
I'm trying to add a new script file to my website but I'm honestly very confused about what I'm supposed to be doing

**Pull request merged**